test/commands-tests/history.js

```diff
@@ -62,6 +62,24 @@ function countHistoryLogs (result) {
     return counter;
 }
 
+async function linkPackages () {
```

Comment:
is this still needed after, you have added the `npm run develop` script in the travis file?

Reply:
yes, it's needed, because installed packages are from 'npm' not from the current branch